[Workflow] Add `getEnabledTransition()` method (#19243)
symfony/symfony-docs: workflow.rst

        }
    }

> To get all enabled transitions of a Workflow, you can use

Author: ```suggestion
To get the enabled transition of a Workflow, you can use
```

> :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransitions`

Author: ```suggestion
:method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition`
```

method.

.. versionadded:: 7.1

>     The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransitions` method

Author: ```suggestion
    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition` method
```

>     The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition` method
>     was introduced in Symfony 7.1.

Reviewer: ```suggestion
    The :method:`Symfony\\Component\\Workflow\\WorkflowInterface::getEnabledTransition`
    method was introduced in Symfony 7.1.
```
